### ISSUE TITLE
add support twoyi openSSL 3.0 fix package manager apt pkg

### ISSUE THREAD

**Issue by ExtinctDrift19** [twoyi/twoyi#72]
Termux package manager not apt pkg working absence OpenSSL certificate

**Comment by tiann** [comment]
Please give me the step to reproduce

**Comment by tiann** [comment]
termux supported in 0.4.0, please retry it.

**Comment by tiann** [comment]
Fixed in 0.7.0 https://coolapk.com/apk/io.twoyi

**Comment by ItsIgnacioPortal** [comment]
The page does not load :(

```
您的访问不匹配网站管理员设置的安全策略，可能会对网站造成危险，已被管理员设置拦截！

可能原因： 访问中带有不合法的参数

如何解决：

1）如果你认为这是一个正常的访问请求，请联系网站管理员解决此问题。

2）如果你是网站管理员，请你调整腾讯云安全加速的策略；前往安全加速控制台。
```
